include/aspect/simulator_access.h

```diff
@@ -51,14 +51,18 @@ namespace aspect
 {
   using namespace dealii;
 
-  // forward declaration
+  // forward declarations:
   template <int dim> class Simulator;
   template <int dim> struct SimulatorSignals;
   template <int dim> class LateralAveraging;
   namespace HeatingModel
   {
     template <int dim> class Manager;
   }
+  namespace AdiabaticConditions
+  {
+    template <int dim> class Interface;
```

Comment:
You are right. I did not think of that because all other interfaces are not derived from SimulatorAccess. Ok then.